docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1098,6 +1099,144 @@ team can more easilly read the messages.
 It is also possible to configure the persistence properties for a specific tenant, for example to set an alternate
 database hostname or database schema name.
 
+## 4.10 Bulk data operations
+### 4.10.1 Bulk data export
+Bulk data export is implemented according to the [HL7 FHIR BulkDataAccess IG: STU1](http://hl7.org/fhir/uv/bulkdata/STU1/export/index.html).
+There are 2 modules involved inside the implementation:
+- fhir-operation-bulkdata
+- fhir-bulkimportexport-webapp   
+
+To integration test, there are tests in ExportOperationTest.java in fhir-server-test module with server integration test cases for system, patient and group export.  
+The *fhir-operation-bulkdata* project implements the REST APIs for bulk data export as FHIR operations.  There are three operations:
+* ExportOperation - system export
+* PatientExportOperation - Patient export
+* GroupExportOperation - group export.
+Each operation calls the JavaBatch framework defined in the *fhir-bulkimportexport-webapp* project to execute the export unit-of-work.   
+There are 3 chunk style JavaBatch jobs defined as following in *fhir-bulkimportexport-webapp* project for the above 3 export operations:  
```

Comment:
```suggestion
There are 3 chunk style JavaBatch jobs defined as following in *fhir-bulkimportexport-webapp* module for the above 3 export operations:  
```

Reply:
just noticed that it's changed to project by Lee.